quickwit-metastore/src/tests.rs

```diff
@@ -1996,6 +1996,7 @@ pub mod test_suite {
             .unwrap();
```

Comment:
Let's add a test that checks that the publish timestamp is not updated when a split is re-published.